webpack.release.config.js

```diff
@@ -43,7 +43,7 @@ entryConfig.plugins = plugins({ isProduction: false, analyzerFile: 'okta-sign-in
 
 // 2. cdnConfig
 var cdnConfig = config('okta-sign-in.min.js');
-cdnConfig.entry.unshift('babel-polyfill');
+cdnConfig.entry.unshift('idempotent-babel-polyfill');
```

Comment:
probably need to update the `devConfig` below as well.

Reply:
Whoops, yep. I guess I should also commit the yarn.lock and keep babel-polyfill in the dev dependencies, since it's used in the karma config.

Reply:
Hmm, after doing that it looks like karma still runs the tests (verified by inspecting in Chrome), but doesn't output results to the terminal.

Btw, another option might be to move to Babel v7+ and use the `noConflict` mode: https://github.com/babel/babel/blob/master/CHANGELOG.md#rocket-new-feature-12